vllm/model_executor/models/llama.py

```diff
@@ -143,10 +146,16 @@ class LlamaDecoderLayer(nn.Module):
     def __init__(self, config: LlamaConfig):
         super().__init__()
         self.hidden_size = config.hidden_size
+        # Requires transformers > 4.32.0
+        try:
+            rope_theta = config.rope_theta
+        except AttributeError:
+            rope_theta = 10000
```

Comment:
Nit: Maybe this can be simplified?
```suggestion
        rope_theta = getattr(config, "rope_theta", 10000)
```

Reply:
Sure, we can do that!